### PR TITLE
27 square carousel

### LINKED ISSUE #27: SquareCarousel
## Component Type: Stateless

Check each box once you have completed done the task.

-   [ ] Component created
-   [ ] Styles written
-   [ ] Storybook created, with stories for all normal states
-   [ ] Tests written, with snapshot test at minimum
-   [ ] Code reviewed

Add PR link here when ready:

### PR DESCRIPTION
**Pull request by S-unya**
**Ensure each of these tasks is complete before merging**

Use component name as PR title and request code review before merging. You can
open a PR and request review for an incomplete component if you'd like.

-   [x] Component created
-   [x] Styles written
-   [x] Storybook created, with stories for all normal states
-   [-] Tests written, with snapshot test at minimum

Closes #27

### CODE REVIEW

**Comment by S-unya** [comment]
There is scope for more testing in `useSlidingCarousel`, but I have written tests for the key functions

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": [
+        "@aerian/eslint-config-react",
+        "@aerian/eslint-config-typescript"
+    ],
+    "rules": {
+        "jsx-a11y/heading-has-content": 0,
+        "complexity": ["warn", 6]
```
Comment (ascorbic) [inline .eslintrc]: This is now the default

**Comment by ascorbic** [comment]
This includes the TweetCard branch, right? It can't really be merged before that one in that case.

File: .eslintrc (hunk 1 above)
Comment (S-unya) [reply .eslintrc]: That was restrained of you! I imagined you'd have put it up to 8.

**Comment by S-unya** [comment]
Yes I merged TwitterCard in. What more needs doing on TwitterCard?

**Comment by ascorbic** [comment]
It's the issue with having `useStaticQuery` in storybooks. I'm still at a loss to see how to handle that, as my suggestion at https://github.com/gatsbyjs/gatsby/issues/18449 doesn't seem to be going anywhere right now

**Pull request merged**